### DISCUSSION TITLE
android app for paperless

### DISCUSSION THREAD

**Discussion by Leen58** [bauerj/paperless_app#83] [Q&A] [answered]
![Screenshot_20220506-121139](https://user-images.githubusercontent.com/70329246/167153441-5bc51f81-16e2-4478-bf80-03abc2f92774.jpg)
I cannot login with the android paperless app in my synology docker paperless ngx server. I can login through my webbrowser on my laptop with the same credentials. I attached a screenshot with the many error messages.
Csan anyonetell me what i am doing wrong?

**Comment by benedetto** [comment]
Exactly the same over here. I would be highly interested in a solution

**Comment by bauerj** [comment]
Sorry, it seems I missed this discussion.

This message usually means that there's a misconfigured reverse proxy between the app and paperless. If that's the case please post your current configuration here and we'll have a look.

**Comment by benedetto** [comment, marked as answer]
In my case, there's no reverse proxy involved. Just a paperless-ngx running on one machine in my network and the app on the phone in the same network.
But I guess I got my error: on the phone I have the app blokada as a system wide ad blocker. Since blokada emulates a VPN to route all the phone's traffic through it, it seems to have some issues with resolving local network addresses... Switching off blokada while I try to access paperless seems to do the trick.

**Comment by Leen58** [comment]
I am using synology ddns with letsencrypt certificate. This is un
trusted however. In docker the configured port is 8777. so intheapp i use
https://lsprbv2.diskstation.me:8777 and this gives the before mentuonned
error.



Op zo 22 mei 2022 11:59 schreef Johann Bauer ***@***.***>:

**Reply by bauerj** [reply]
When I open that URL in my browser I get an error as well. Are you sure this is not the HTTP port? Maybe try http://lsprbv2.diskstation.me:8777/ if you want an unencrypted connection.
